Rakefile

```diff
@@ -12,10 +12,22 @@ solr_home_path = 'solr_configs'
 SolrWrapper.default_instance_options = {
     verbose: true,
     port: '8888',
-    version: '7.1.0',
+    version: '8.1.0',
     instance_dir: solr_instance_path,
     download_dir: solr_download_path,
     solr_options: {'s' => solr_home_path}
 }
 
 require 'solr_wrapper/rake_task'
+
+namespace :solr do
+  task :test_wait do
```

Comment:
Perhaps we should delete this if we can't get it to create the cores. I looked into it a bit; seems like it must be related to that `solr_options: {'s' => solr_home_path}` but I couldn't figure out how that works. 

There is a `rake solr:stop` task that works for me.